packages/wonder-blocks-timing/util/animation-frame.js

```diff
@@ -25,6 +25,10 @@ export default class AnimationFrame implements IAnimationFrame {
      * @memberof AnimationFrame
      */
     constructor(action: () => mixed, autoSet?: boolean) {
+        if (typeof action !== "function") {
```

Comment:
For learning purposes, is there a way in `flow` to define that `action` has a function signature and avoid using `typeof` to check this?

Reply:
That is already done, but `flow` is only a code-time check, not a runtime check. Not everyone that consumes our code is necessarily going to be using flow or type checking. Since this is a public facing API, I want it to be robust.

Reply:
That's an excellent point! thanks for clarifying.